src/components/HomePage/HomePage.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import WelcomeUser from './WelcomeUser'
+import TopLeftCard from '../Design/CardWrapper/TopLeftCard'
+import NavButton from './NavButton'
+
+export default function HomePage({ user }) {
+  return (
+    <div>
```

Comment:
I think a fraction instead of a div would be enough unless you still want to style it later.

Reply:
Thanks for your suggestions! :)